tendrl/commons/objects/__init__.py

```diff
@@ -28,6 +28,7 @@ def thread_safe_method(self, *args, **kws):
 class BaseObject(object):
     def __init__(self, *args, **kwargs):
         self._ttl = None
+        self._ttl_attr = []
```

Comment:
Lets rename this attribute as `self._attrs_with_ttl`, looks more meaningful.

Reply:
ok